sheetfu/modules/table.py

```diff
@@ -134,18 +155,17 @@ def get_full_table_syncro_batches(self):
             table_backgrounds.append(item_backgrounds)
             table_font_colors.append(item_font_colors)
 
-        self.items_range.set_values(table_values, batch_to=self)
-        if self.has_notes:
-            self.items_range.set_notes(table_notes, batch_to=self)
-        if self.has_backgrounds:
-            self.items_range.set_backgrounds(table_backgrounds, batch_to=self)
-        if self.has_font_colors:
-            self.items_range.set_font_colors(table_font_colors, batch_to=self)
+        self._generate_set_own_range_values_batches(range=self.items_range, values=table_values, notes=table_notes,
+                                                    backgrounds=table_backgrounds, font_colors=table_font_colors)
+
+    def _generate_delete_intitial_items_range_batch(self):
+        if self.items_range is None:
+            return
+        empty_column_values = [""] * self.items_range.coordinates.number_of_columns
+        empty_values = [empty_column_values] * self.items_range.coordinates.number_of_rows
```

Comment:
This is fine for this application, but be careful when doing `*`operations with lists.

Here's why: [Link](https://stackoverflow.com/a/3459140/918055)

Reply:
Interesting read!